numberformat/proposed.emu

```diff
@@ -4,6 +4,62 @@
   <emu-clause id="sec-numberformat-abstracts">
     <h1>Abstract Operations for NumberFormat Objects</h1>
 
+    <emu-clause id="sec-runtime-semantics-stringintlmv" type="sdo">
+      <h1>Runtime Semantics: StringIntlMV</h1>
```

Comment:
@sffc @bakkot @ryzokuken I defined the StringIntMV as a new interpretation used in [ToIntlMathematicalValue](https://github.com/tc39/proposal-intl-numberformat-v3/pull/82/files#diff-4257dcc2ffce97f70f5c4a8340725215fe949c64cac4ffd601e069544657e45bR1056). 

This was the recommendation @bakkot ? 

There few things to adjust on this new StringIntMV , it's not returning the RoundMVResult but have to respect the +∞, -∞, NaN, and -0 in ToIntlMathematicalValue.

But just want to check if this it's the correct approach